.readthedocs.yaml

```
@@ -0,0 +1,7 @@
+version: 2
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.10"
```

Using the latest-stable version would be better IMO.

```suggestion
    python: "3"
```